src/silx/gui/plot/PlotInteraction.py

```diff
@@ -1636,14 +1620,15 @@ def getDescription(self):
         return params
 
 
-class PlotInteraction(object):
-    """Proxy to currently use state machine for interaction.
-
-    This allows to switch interactive mode.
+class PlotInteraction(qt.QObject):
```

Comment:
This is added to the API as it can be retrieved through `PlotWidget.interaction()`, so some methods are made private.